packages/core/test/actionCreators.test.ts

```diff
@@ -17,10 +16,13 @@ describe('action creators', () => {
     });
 
     it('should accept an event object', () => {
-      const action = send({ type: 'foo', bar: 'baz' });
+      const action = raise({
+        type: 'foo',
+        // @ts-ignore
+        bar: 'baz'
+      });
```

Comment:
It could be changed like this:
```suggestion
      const event = {
        type: 'foo' as const,
        bar: 'baz'
      };
      const action = raise<{}, typeof event>(event);
```

Right now we can't avoid explicit type params here, the types are more fine-tuned for nested inference than for creating those action objects standalone. We can explore fixing this in the future but this was the only way to fix the current inference that I was able to find.